Mathlib/Algebra/CharZero/Defs.lean

```diff
@@ -93,6 +93,9 @@ end Nat
 
 namespace OfNat
 
+-- TODO: This whole section is draped in `no_index` because of its special casing in
+-- core, see leanprover/lean4#2867. Hopefully this can be fixed in the future.
+
```

Comment:
The standard comment we use (on every lemma) is `-- See note [no_index around OfNat.ofNat]`